pkg/controller.v1/tensorflow/controller.go

```diff
@@ -70,6 +70,9 @@ const (
 	PodEvictAnnotation   = "cluster-autoscaler.kubernetes.io/safe-to-evict"
 
 	TFJobWaitingWorkerAnnotation = "arena.kubeflow.org/pod.ttlSecondsAfterFinished"
+
+	// JobPastActiveDeadlineEventAnnotation job past active deadline only report event
+	JobPastActiveDeadlineEventAnnotation = "arena.kubeflow.org/job-past-active-deadline"
```

Comment:
arena.kubeflow.org/job-past-active-deadline to arena.kubeflow.org/job-timeout-action. Default value is fail, and another value is event.

Reply:
Done